stable/redis/templates/redis-master-statefulset.yaml

```diff
@@ -269,25 +269,29 @@ spec:
       - name: metrics
         image: {{ template "redis.metrics.image" . }}
         imagePullPolicy: {{ .Values.metrics.image.pullPolicy | quote }}
+        command:
+        - /bin/bash
+        - -c
+        - |
+          if [[ -f '/secrets/redis-password' ]]; then
+           password_aux-`cat /secret/redis-password`
```

Comment:
You don't need an aux variable here. Also, using `$()` is preferred instead of using backticks.